### ISSUE TITLE
Loading a x64 dll

### ISSUE THREAD

**Issue by gwenzek** [zabirauf/icsharp#20]
Hi, thanks for this integration with ipython, it's really nice.

I can't load dll compiled as 'x64'.
The error I receive is `AssemblyLoader failed to load assembly xxx.xxx, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null `.

I tried with the same dll at the same location compiled in 'x86' and in 'AnyCpu' and it works.

Is the kernel launched in a 32-bit mode ?
Is it possible to launch it in x64 ?

**Comment by gwenzek** [comment]
Okay, I recompiled the Kernel project in x64 mode, and replace the `iCSharp.Kernel.exe` in `AppData\Roaming\icsharp` by the x64 version.
Now I can load my x64 dll :-)

I guess there should be an option in Chocolate to use a x64 build of iCSharp.

**Comment by denfromufa** [comment]
@gwenzek can you compile in anycpu and check x64?

**Comment by gwenzek** [comment]
Ok, I retried in AnyCpu and with the "prefer 32 bit" option I don't see my x64 dlls but without it works fine.
But that was expected since I'm on a x64 machine so AnyCpu will be launched in x64 except when "prefer 32 bit" is on.

I think you should provide installation instruction from the sources and therefore let the user choose if he wants a 32bit or 64bit kernel.

**Comment by zabirauf** [comment]
@gwenzek I agree, we should have a better installation mechanism that should work seamlessly for both x86 and x64.
